packages/client/components/DashFilterToggle/DashFilterToggle.tsx

```diff
@@ -21,8 +25,12 @@ const StyledIcon = styled(Icon)({
 const StyledLinkButton = styled(LinkButton)({
   color: PALETTE.TEXT_GRAY,
   fontWeight: 600,
+  marginRight: '16px',
```

Comment:
While adding margins (which affect layout) here is safe because of the narrow scope of this component, I like to sometimes separate styles for a control from styles for the layout, so that they control can be used in many layouts. I’ll handle the layout in the containing component and context. One way is to make this component styleable by adding the className property (which I did in a separate commit). Another way is to use whatever containing element or wrapper to handle layout.